etc/non-ecs-schema.json

```diff
@@ -8,9 +8,18 @@
     }
   },
   "winlogbeat-*": {
-    "winlog.event_data.OriginalFileName": "keyword",
-    "winlog.event_data.GrantedAccess": "keyword",
-    "winlog.event_data.CallTrace": "keyword",
+    "winlog": {
+      "event_data": {
+        "AccessList": "keyword",
+        "AttributeLDAPDisplayName": "keyword",
+        "AttributeValue": "keyword",
+        "CallTrace": "keyword",
+        "OriginalFileName": "keyword",
+        "GrantedAccess": "keyword",
+        "RelativeTargetName": "keyword",
+        "ShareName": "keyword"
+      }
+    },
     "powershell.file.script_block_text": "text"
```

Comment:
eaf6e21 I nested the existing forma a little more and moved your additions to `logs-system` to `winlogbeat`

Reply:
@brokensound77 I think we need to maintain the logs-system* one... As we need to support Elastic Agent Integration too and not only winlogbeat

Reply:
validation only occurs against beats schemas for non-ECS fields (we may eventually expand to specific integrations)

Reply:
Oh, got it